src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -930,6 +930,35 @@ QString QgsPostgresConn::quotedIdentifier( QString ident )
   return ident;
 }
 
+static QString quotedString( QString v )
```

Comment:
No. It is modified by the `replace` method. It was like that before and had the same reflex. But nope.